lib/pipeline.js

```diff
@@ -69,6 +69,43 @@ class PipelineModel extends BaseModel {
 
         return admin;
     }
+
+    /**
+     * Fetch all jobs that belong to this pipeline
+     * @property jobs
+     * @return Promise
+    */
+    get jobs() {
+        const listConfig = {
+            params: {
+                pipelineId: this.id
+            },
+            paginate: {
+                count: 25, // This limit is set by the matrix restriction
+                page: 1
+            }
+        };
+
+        // Lazy load factory dependency to prevent circular dependency issues
+        // https://nodejs.org/api/modules.html#modules_cycles
+        /* eslint-disable global-require */
+        const JobFactory = require('./jobFactory');
+        /* eslint-enable global-require */
+        const factory = JobFactory.getInstance();
+
+        const jobs = factory.list(listConfig)
+            .then((records) => records.sort((job1, job2) => job1.id - job2.id));
```

Comment:
This sorting doesn't really make sense. Just spit them out unsorted until we save workflow somewhere.